homework-1 (from lesson 2020-09-01)

alexvlife opened the pull request.

alexvlife committed:
homework-1 (from lesson 2020-09-01)

koretskiyav commented on src/components/restaurant.js:
```diff
@@ -0,0 +1,12 @@
+
+export default function Restaurant(props) {
+  return (
+    <div>
```
нет среднего рейтинга

alexvlife replied:
Реализовано. В Restaurant произведен подсчет среднего рейтинга и его отображение с помощью компоненты Rate.
Коммит: 8b9be4d45aed81aa2cfe51d1081785f3543ec79e

alexvlife committed:
fix - homework-1 (from lesson 2020-09-01)

- В Restaurant реализован подсчет среднего рейтинга и его отображение с помощью компоненты Rate;
- внесены корректировки в компонент Reviews - теперь в пропсах принимает непосредственно reviews
